### PR TITLE
Converted product card to use view-form

### PR DESCRIPTION
**Pull request by pozil**

### CODE REVIEW

File: force-app/main/default/lwc/productCard/productCard.js [hunk 1]
```diff
@@ -2,43 +2,29 @@ import { LightningElement, wire } from 'lwc';
...
+/** Utils to extract field values. */
+import { getFieldValue } from 'lightning/uiRecordApi';
 
 /** Pub-sub mechanism for sibling component communication. */
 import { registerListener, unregisterAllListeners } from 'c/pubsub';
 
 /** Product__c Schema. */
 import PRODUCT_OBJECT from '@salesforce/schema/Product__c';
 import NAME_FIELD from '@salesforce/schema/Product__c.Name';
-import LEVEL_FIELD from '@salesforce/schema/Product__c.Level__c';
```
Comment (muenzpraeger) [inline force-app/main/default/lwc/productCard/productCard.js:14]: We should leave the static imports of field names to provide a best practice example with referential integrity, similar to [this](https://github.com/trailheadapps/lwc-recipes/blob/master/force-app/main/default/lwc/recordViewFormStaticContact/recordViewFormStaticContact.js#L10-L15). Thoughts, @kevinv11n?

Comment (kevinv11n) [reply force-app/main/default/lwc/productCard/productCard.js:14]: Referential integrity is recommended. Unless the example is purposely not using it (like https://github.com/trailheadapps/lwc-recipes/tree/master/force-app/main/default/lwc/recordFormDynamicContact) then it should use referential integrity. I think this example falls into the case of using it. 

`<lightning-record-view-form>` supports receiving the objects imported from `@salesforce/schema`. See https://github.com/trailheadapps/lwc-recipes/blob/master/force-app/main/default/lwc/recordViewFormStaticContact/recordViewFormStaticContact.js#L10-L15

File: force-app/main/default/lwc/productCard/productCard.js [hunk 2]
```diff
@@ -48,6 +34,18 @@ export default class ProductCard extends NavigationMixin(LightningElement) {
         unregisterAllListeners(this);
     }
 
+    handleRecordLoaded(event) {
+        const { records } = event.detail;
+        if (records[this.recordId]) {
```
Comment (kevinv11n) [inline force-app/main/default/lwc/productCard/productCard.js:39]: I'm surprised this check is required

Comment (pozil) [reply force-app/main/default/lwc/productCard/productCard.js]: Good point, I removed it.

File: force-app/main/default/lwc/productCard/productCard.js (hunk 1 above)
Comment (pozil) [reply force-app/main/default/lwc/productCard/productCard.js:14]: Thanks for the review, I added the field references.

**Pull request merged**